### ISSUE TITLE
Cannot open D:\Users\USER\Anaconda3\Scripts\jupyter-script.py while using any jupyter commands in anaconda prompt

### ISSUE THREAD

**Issue by soumya997** [jupyterlab/jupyterlab#9797]
## Description

I was trying to install some extensions for jupyterlab using `jupyter` command. but after executing that `jupyter` command for the respective extension it was giving me a message which is like this,
`Cannot open D:\Users\USER\Anaconda3\Scripts\jupyter-script.py`
here is the screenshot 
<br>
<img src="https://i.ibb.co/pR81zLT/Screenshot-967.png" width="600">

## Reproduce

I don't know why it's happening, so I can't really give any instructions for reproducing the problem.

## Expected behavior

It will throw this line -> `Cannot open D:\Users\USER\Anaconda3\Scripts\jupyter-script.py`

## Context

- Operating System and version: windows 10
- Browser and version: Version 88.0.4324.150 (Official Build) (64-bit)
- JupyterLab version:  jupyterlab-3.0.7

<details><summary>Troubleshoot Output</summary>
<pre>
1. I tried reinstalling jupyterlab, but it did not helped. showed the same message.
2. It seems like jupyterlab 1.0.2 and  jupyterlab-3.0.7 having the same issue. showed the same message.

</pre>
</details>

**Comment by abaaqi** [comment]
Hello, did you later get a solution to this? I am facing the same issue now just after installing git extension

**Comment by soumya997** [comment]
Nope, I tried multiple things but last I opened it was showing "404: not found". Jupyter lab does not work anymore in my system. You may need to try reinstalling anaconda.

**Comment by abaaqi** [comment]
Thank you @soumya997. I later found a solution. In case anyone else comes here in search of a solution for this. Here is what I did and I hope it works for you too. In Anaconda Navigator, I clicked on the settings icon here and then removed jupyter lab. Then I installed it back after removing it and voila, I have my jupyter lab back.
![image](https://user-images.githubusercontent.com/85158657/148062520-fa2bcbe8-a8d0-4abc-8159-b58bde72d89f.png)

**Comment by soumya997** [comment]
Holy cow, it worked. Thank you so much @abaaqi . I forgot that there is also anaconda navigator too😂.
